src/doc/src/reference/environment-variables.md

```diff
@@ -184,6 +184,8 @@ let version = env!("CARGO_PKG_VERSION");
 * `CARGO_PKG_DESCRIPTION` — The description from the manifest of your package.
 * `CARGO_PKG_HOMEPAGE` — The home page from the manifest of your package.
 * `CARGO_PKG_REPOSITORY` — The repository from the manifest of your package.
+* `CARGO_CRATE_NAME` — The name of the crate that is currently being compiled.
+* `CARGO_BIN_NAME` — The name of the binary that is currently being compiled (if it is a binary).
```

Comment:
Maybe this should clarify that on Windows the name does *not* contain the `.exe` extension?

Reply:
Adding "This name does not include any file extension, such as `.exe`." would work.